nav2_straightline_planner/src/nav2_straightline_planner.cpp

```diff
@@ -4,6 +4,7 @@
 #include "nav2_util/node_utils.hpp"
 
 #include "nav2_straightline_planner/nav2_straightline_planner.hpp"
+#include "nav2_straightline_planner/rrt_planner.hpp"
```

Comment:
this should be its own package, i.e you can rebuild this package from the ground up renaming everything to `nav2_rrt_planner_plugin`